### PR TITLE
Finished assignment1

### PR DESCRIPTION
**Pull request by marcwoodburysmith**
Slack username:

### CODE REVIEW

File: main.cpp [hunk 1]
```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
...
-
-//  action 3:
-
+//  Noun:    human
+//  action 1:    human laughs
+human.laugh();
+//  action 2:    human cries
+human.cry();
+//  action 3:    human gasps
+human.gasp();
```
Comment (matkatmusic) [inline main.cpp]: As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [x] come up with a new noun and 3 actions
- [x] pick a noun that needs `camelCase` for the name
- [x] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.

File: main.cpp [hunk 2]
```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
...
-
-//  action 3:
-
+//  Noun:    engine
+//  action 1:    engine accelerates
+engine.accelerate();
+//  action 2:    engine decelerates
+engine.decelerate();
+//  action 3:    engine neutral
+engine.neutral();
```
Comment (matkatmusic) [inline main.cpp]: `neutral` is not an action.  You need to include a verb in the action phrase.
- [ ] revise

**Review by matkatmusic — CHANGES_REQUESTED** [review]

File: main.cpp (hunk 1 above)
Comment (marcwoodburysmith) [reply main.cpp]: Created new noun and actions as requested.

File: main.cpp (hunk 2 above)
Comment (marcwoodburysmith) [reply main.cpp]: Replaced with verb.

**Review by matkatmusic — APPROVED** [review]

**Pull request merged**